pkgs/development/libraries/science/math/scs/default.nix

```diff
@@ -4,6 +4,7 @@ stdenv.mkDerivation rec {
   pname = "scs";
   version = "2.1.1";
 
+  # NOTE: check pythonPackages.python-scs when updating.
```

Comment:
this is likely uneeded, most reviewers will do `nix-review` which will pick up that this derivation has changed.

at the very least it should go in a different commit